Makefile

```diff
@@ -13,6 +13,7 @@ COMPOSE          = \
     -f docker-compose.cypress.yml \
     -f docker-compose.edx.yml \
     -f docker-compose.keycloak.yml \
+    -f docker-compose.utils.yml \
```

Comment:
Is it necessary to create a new docker-compose file for this service? Why not declaring it in the main docker-compose file ?

Reply:
Indeed, it would be preferable to add the service to an existing docker-compose) Thanks) I have moved it to the `docker-compose.cypress.yml`